app/controllers/SummaryDeclarationController.scala

```diff
@@ -61,8 +62,16 @@ trait SummaryDeclarationController extends ERSReturnBaseController with Authenti
       if (reportableEvents == PageBuilder.OPTION_YES) {
         fileType = all.getEntry[CheckFileType](CacheUtil.FILE_TYPE_CACHE).get.checkFileType.get
         if (fileType == PageBuilder.OPTION_CSV) {
-          val csvFilesCallback: List[CsvFilesCallback] = all.getEntry[CsvFilesCallbackList](CacheUtil.CHECK_CSV_FILES).get.files
-          for (file <- csvFilesCallback if file.callbackData.isDefined) {
+          val csvCallback = all.getEntry[UpscanCsvFilesCallbackList](CacheUtil.CHECK_CSV_FILES).get
```

Comment:
use of .get here. There seems to be a bit of repeating code here also that could be refactored.

Reply:
Will change the exception thrown here as an exception is the desired behaviour
There seems to be a bigger piece of work here to address all the .gets as there's another two on the lines above too. I wont address that in this ticket